website/docs/reference/activation-strategies.md

```diff
@@ -2,130 +2,167 @@
 
-- [Standard](#standard)
-- [UserIDs](#userids)
-- [Gradual Rollout](#gradual-rollout)
-- [IPs](#ips)
-- [Hostnames](#hostnames)
+Feature flags can have multiple activation strategies. Unleash evaluates each strategy independently, enabling the flag if any resolves to true. This behavior is equivalent to the OR logical operator.
 
-## Standard {#standard}
+For example, to roll out a feature to 75% of users while granting access to internal users, you can add two activation strategies as follows: 
+1. Gradual rollout to 75%.
+2. Gradual rollout to 100% with a constraint on the email address.
+   
+![A feature flag with two strategies](/img/activation-strategies-example.png)
 
-A basic strategy that means "active for everyone".
+## Rollout percentage
 
-This strategy has the following modelling name in the code:
+The rollout percentage determines the proportion of users exposed to a feature. It uses a normalized [MurmurHash](https://en.wikipedia.org/wiki/MurmurHash) of a user’s unique ID, ensuring consistent and random feature distribution. [Stickiness](/reference/stickiness) maintains a stable user experience across sessions.
 
-- **default**
+## Targeting
 
-## UserIDs {#userids}
+Segmentation and constraints allow you to define conditions for your activation strategies so that they will only be evaluated for users and applications that match those criteria. Constraints are individual conditional statements, while [segments](/reference/segments) are a reusable set of constraints that you can apply to multiple strategies.
 
-Active for users with a `userId` defined in the `userIds` list. A typical use case is to enable a feature for a few specific devs or key persons before enabling the feature for everyone else. This strategy allows you to specify a list of user IDs that you want to expose the new feature for. (A user id may, of course, be an email if that is more appropriate in your system.)
+### Constraints
 
-**Parameters**
+:::note Availability
 
-- userIds - _List of user IDs you want the feature flag to be enabled for_
+**Version**: `4.16+`
 
-This strategy has the following modelling name in the code:
+:::
 
-- **userWithId**
+Constraints are conditional rules that determine whether a strategy applies, based on fields from the [Unleash context](/reference/unleash-context). Constraints can reference both [standard context fields](../reference/unleash-context#overview) and [custom context fields](../reference/unleash-context#custom-context-fields).
 
-## Gradual Rollout {#gradual-rollout}
+An activation strategy can have as many constraints as needed. When an activation strategy has multiple constraints, then every constraint must be evaluated to true for the strategy to be evaluated. This behavior is equivalent to the AND logical operator. 
```

Comment:
Maybe adding a link to playground saying that you can experiment constraints with it